kvmagent/kvmagent/test/test_parsing_multipath.py

```
@@ -26,35 +35,35 @@ def test_write_conf():
     # we configure our default device, which will override features "1 Queue"_ if_ no_ path"， no_ path_ Retry 60,
     # change to feature 0 no_ path_ retry fail
-    assert multipath.write_multipath_conf('multipath1.conf', blacklist)
-    assert not multipath.write_multipath_conf('multipath1.conf', blacklist)
+    assert write_multipath_conf('multipath1.conf', blacklist)
+    assert not write_multipath_conf('multipath1.conf', blacklist)
     with open("multipath1.conf",
          "r") as fd:
         text3 = multipath.parse_multipath_conf(fd)
     with open("multipath1.conf.out", "r") as fd1:
         text4 = multipath.parse_multipath_conf(fd1)
     assert cmp(text3, text4) == 0
```

Ruff (0.12.2)

Undefined name `cmp`

(F821)